prime-dotnet-webapi/Services/DefaultEnrolmentCertificateService.cs

```diff
@@ -44,6 +51,9 @@ public async Task<EnrolmentCertificate> GetEnrolmentCertificateAsync(Guid access
 
         public async Task<EnrolmentCertificateAccessToken> CreateCertificateAccessTokenAsync(Enrollee enrollee)
         {
+            // Add privileges to Enrollee
+            enrollee.Privileges = await _privilegeService.GetPrivilegesForEnrolleeAsync(enrollee);
+
```

Comment:
The privileges are not used to create the token, we don't need these lines here.

Reply:
kk!